skietbaan_fe/src/components/Groups.js

```diff
@@ -1,117 +1,179 @@
-              <td ><h5>{post.username} {index}</h5>
-                <h5>{post.email}</h5></td>
-            </tr>
-          ))}
-        </tbody>
-      </Table>
+    const postitems = (
+      <div className="check">
+        <ul className="list-group">
+          {this.state.posts
+            .filter(post => {
+              return (
+                !this.state.filterText ||
+                post.username
+                  .toLowerCase()
+                  .startsWith(this.state.filterText.toLowerCase()) ||
+                post.email
+                  .toLowerCase()
+                  .startsWith(this.state.filterText.toLowerCase())
+              );
+            })
+            .map((post, index) => (
+              <li className="list-group-item list-group-item-light" key={post.id}>
+                <input
+                  type="checkbox"
+                  className="boxs"
+                  onClick={() => this.toggleHighlight(index)}
+                  checked={post.highlighted}
+                />
+                <label className="blabe">
+                  <div className="userName">{post.username}</div>
+                  <div className="emails">{post.email}</div>
+                </label>
+              </li>
+            ))}
+        </ul>
+      </div>
     );
     return (
-      <main>
-        <div main className="topnav" data-simplebar data-simplebar-auto-hide="false">
-          <label className="create">Create Groups</label>
-         
-          <label className="label" onChange={this.handleOnChange}>{this.state.count}</label>
-          <br />
-          <a className="search" href="/Groups"><span class="glyphicon glyphicon-chevron-left"></span></a>
-          <input className="texts" type="text" placeholder="Search.." />
+      <main className="TheMain">
+        <div className="TheNavBar">
+          <a href="#" className="fa fa-angle-left" onClick={this.onBack} />
+
+          <h2 className="center_label">
+            {this.props.name}
+          </h2>
         </div>
-        <div className="middle">
-          <label className="b"><b>Add Members</b></label>
-          <button className="add" onClick={this.handleOnClick}>Add all</button>
-       </div>
-        <br />
-        <br />
-        <div className="scrollbar" data-simplebar data-simplebar-auto-hide="false">
-          {postItems}
+        <div className="BNavBar">
+          <input
+            className="theText"
+            id="username"
+            type="text"
+            onChange={this.onChange}
+            autoComplete="off"
+          />
+          <button
+            className={this.state.count == 0 ? "select" : "select2"}
+            id="check"
+            onClick={this.selectall}
+          >
+            {this.state.check}
+          </button>
         </div>
+
+        <div className="OnToTheNextOne" />
```

Comment:
Inconsistent class names choose camelCase or CapitalCase. I suggest camelCase. This needs to reflect in all class names